core/foundation/src/TClassEdit.cxx

```diff
@@ -1274,15 +1284,8 @@ int TClassEdit::GetSplit(const char *type, vector<string>& output, int &nestedLo
 
 string TClassEdit::CleanType(const char *typeDesc, int mode, const char **tail)
 {
-   static const char* remove[] = {"class", "const", "volatile", nullptr};
-   auto initLengthsVector = []() {
-      std::vector<size_t> create_lengths;
-      for (int k=0; remove[k]; ++k) {
-         create_lengths.push_back(strlen(remove[k]));
-      }
-      return create_lengths;
-   };
-   static std::vector<size_t> lengths{ initLengthsVector() };
+   constexpr std::array<const char *, 3> remove{"class", "const", "volatile"};
+   constexpr auto lengths = getLengthsArray(std::make_index_sequence<std::size(remove)>(), remove);
```

Comment:
This should also work
```cpp
constexpr static auto lengths = []{
    constexpr std::array<std::size_t, 3> ret{};
    for(std::size_t i = 0; i < remove.size(); i++)
        ret[i] = remove[i];
    return ret;
}();
```

Reply:
Explored here: https://github.com/root-project/root/pull/18757